src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
+    return !(
+      document.fullscreenEnabled === false ||
+      document.webkitFullscreenEnabled === false ||
+      document.mozFullScreenEnabled === false ||
+      document.msFullscreenEnabled === false
+    )
```

Comment:
Actually, we had an idea some time ago to maximize the player in the window viewport if the fullscreen API is not available. This way we can support maximizing the player size on some mobile browsers that do not support fullscreen. Perhaps we could create an issue to discuss this idea. What do you think, @tjenkinson?

In the meantime, we can avoid having false positives assuming `undefined` values as unavailable.